modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,53 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+.Procedure
+
+. After creating the Argo CD instance, login into the keycloak console.
++
+[source,yaml]
+----
+kubectl -n argocd get route keycloak
+
+NAME   HOST/PORT                                                                PATH   SERVICES   PORT    TERMINATION   WILDCARD
+keycloak    keycloak-default.apps.ci-ln-******.origin-ci-int-aws.dev.**.com          keycloak        <all>   reencrypt     None
+----
+. Get the Keycloak credentials which is stored as environment variables in the keycloak pod.
++
+[source,yaml]
+----
+  kubectl -n argocd get pods
+
+  NAME                                         READY   STATUS             RESTARTS   AGE
+  keycloak-1-2sjcl                                  1/1     Running            0          45m
+----
+. Get the Keycloak username.
++
+[source,yaml]
+----
+  kubectl -n argocd exec keycloak-1-2sjcl -- "env" | grep SSO_ADMIN_USERNAME
+
+SSO_ADMIN_USERNAME=Cqid54Ih
+----
+. Get the Keycloak password.
++
+[source,yaml]
+----
+  kubectl -n argocd exec keycloak-1-2sjcl -- "env" | grep SSO_ADMIN_PASSWORD
+
+SSO_ADMIN_PASSWORD=GVXxHifH
+----
+. Enter the keycloak username and password. Click *Log in*.
+
+[NOTE]
+====
+Keycloak instance takes 2-3 minutes to be up and running. You will see the option *LOGIN VIA KEYCLOAK* only after the keycloak instance is up.
+====
+
+By default any user logged into Argo CD will have read-only access. You can manage the user level access by updating the `argocd-rbac-cm` configmap.
```

Comment:
Missing comma and avoid future tense when possible (though future tense does read fine here):

By default, any user logged into Argo CD has read-only access.`